bigbluebutton-html5/imports/ui/components/chat/alert/container.jsx

```diff
@@ -54,9 +63,39 @@ const ChatAlertContainer = (props) => {
     })
     : null;
 
+  const chatsTracker = {};
+
+  if (usingChatContext.chats) {
+    const chatsActive = Object.entries(usingChatContext.chats);
+    chatsActive.forEach((c) => {
+      if (c[0] === idChat || (c[0] === 'MAIN-PUBLIC-GROUP-CHAT' && idChat === 'public')) {
+        chatsTracker[c[0]] = {};
+        if (c[1]?.posJoinMessages || c[1]?.messageGroups) {
+          const m = Object.entries(c[1]?.posJoinMessages || c[1]?.messageGroups);
+          chatsTracker[c[0]].count = m?.length;
+          if (m[m.length - 1]) {
+            chatsTracker[c[0]].content = m[m.length - 1][1]?.message;
```

Comment:
Hello @KDSBrowne! Great work, it's working as expected! Just one thing, could you please clarify why that snippet of code is necessary? It seems useless (please, correct me if I'm wrong).

Reply:
Also all these array accessing by index make me a bit nervous :worried: would a try-catch or alternative safety mechanism be applicable in your opinion?

Reply:
Just checking that `usingChatContext.chats`, `c[1]?.posJoinMessages`, `c[1]?.messageGroups` are actually `object`'s does the job.

Something like:
`typeof usingChatContext.chats === 'object'`

Reply:
@JoVictorNunes You are correct that the line of code is useless at the moment because we do not use it yet to show the toast notification. It will be needed though in order to track the message to display.